web-ui/src/pages/__snapshots__/AnniversaryReportPage.test.jsx.snap

```diff
@@ -138,30 +138,6 @@ exports[`renders correctly 1`] = `
         />
       </button>
     </div>
-    <div
```

Comment:
This snapshot file was updated to remove the row of checkboxes that we no longer render.